src/plugins/orangehrmClaimPlugin/Api/EmployeeClaimRequestAPI.php

```diff
@@ -347,7 +388,9 @@ protected function setEmpNumbers(ClaimRequestSearchFilterParams $claimRequestSea
         );
 
         if (!is_null($empNumber)) {
-            if (!$this->getEmployeeService()->getEmployeeDao()->getEmployeeByEmpNumber($empNumber) instanceof Employee) {
+            if (!$this->getEmployeeService()->getEmployeeDao()->getEmployeeByEmpNumber(
+                $empNumber
+            ) instanceof Employee) {
                 throw $this->getRecordNotFoundException();
```

Comment:
@ManulMax do we need to handle this as 404 as this is `getAll`. check similar places

Reply:
No, We handled this in getOnes.